app/institutions/dashboard/-components/institutional-dashboard-wrapper/styles.scss

```diff
@@ -12,14 +12,25 @@
 }
 
 .banner {
+    @include clamp-width;
     padding: 15px 0;
     display: flex;
     align-items: center;
     justify-content: space-between;
+}
 
-    div {
-        color: $color-text-gray-blue;
-    }
+.download-button {
+    display: inline-block;
+    border: 1px solid $color-border-gray;
+    border-radius: 2px;
+    padding: 6px 12px;
+}
+
+.download-button:focus,
+.download-button:hover {
+    color: $color-text-black;
+    background-color: $color-shadow-dark;
+    border-color: $color-border-gray-darker;
```

Comment:
(nitpick) is there a reusable component with style that applies here? (... i guess while `Button` has a "fake link" mode, `OsfLink` doesn't have a "fake button" mode... )

Reply:
Yea, this "OsfLink that looks like a button" is a pattern that we've used before. I guess I could just make this a `Button` that calls a `window.open` and that might be cleaner to maintain.

Reply:
`<a href="...">` is preferable over a `<button>` with javascript to open a url (for both accessibility and clarity) -- if it's a common pattern to style a link as a button, tho, would recommend putting that consistent style in one place (whether `OsfLink` with corresponding arg, or in a reusable global css class or scss whatchamadoo) rather than rewriting it each time

Reply:
Re: `<a>` vs `<button>`, that's a good point. This is opening a URL after all.

I will add an arg to `<OsfLink>` similar to what we do for `<Button>`